src/utils/wallet.js

```diff
@@ -328,27 +326,46 @@ class Wallet {
         return null
     }
 
-    async signInWithLedger() {
+    async getLedgerAccountIds() {
         const publicKey = await this.getLedgerPublicKey()
         await setKeyMeta(publicKey, { type: 'ledger' })
-        const accountIds = await getAccountIds(publicKey.toString())
+        return (
+            (await Promise.all(
+                (await getAccountIds(publicKey.toString()))
+                    .map(async (accountId) => 
+                        await this.getAccount(accountId).findAccessKey()
```

Comment:
it's best to check for specific `publicKey` above explicitly vs using `.findAccessKey()` (which isn't public and so not even guaranteed to stay as is)

Reply:
Also on a second thought – it's likely better not to do filtering here at all. Instead just handle errors when recovering accounts and skip the ones which fail. There can be other reasons for failure (e.g. not enough tokens for storage) that will benefit from this.

Reply:
In this particular case user disabled Ledger for this account earlier, that's why it is checked here, so the account will not be shown in the accounts list in modal - it might be confusing for user to see account that was already disabled.

Reply:
@marcinbodnar still need to change to not use `.findAccessKey()`

Reply:
@vgrichina done